tests/acceptance/features/apiGraph/assignRole.feature

```diff
@@ -0,0 +1,33 @@
+@api
+Feature: assign role
+  As an admin, I want to assign roles to users.
+  I cannot change my own role.
+  Users without an admin role cannot get the list of roles, assignments list and assign roles to users
+
+
+  Scenario Outline: assign role to the user using graph api
+    Given user "Alice" has been created with default attributes and without skeleton files
```

Comment:
How about creating a user using the graph API?

Reply:
isn't the user created through this step is created with graph api?

Reply:
The step creates the user using the Graph API

Reply:
@SwikritiT Yeah, this step works for both provisioning and graph, AFAIK we are using graph API only for users-related CRUD in Ocis. This step should be cleaned up in the near future. Also, we have a dedicated graph API step.
Decision is yours @amrita-shrestha

Reply:
Yes, the step needs to be refactored but that shouldn't be in this PR. This should actually  be a part of https://github.com/owncloud/ocis/issues/5337 or we can create separate issue